admin/jobs/anonymise_expired_applications.py

```diff
@@ -4,13 +4,15 @@
 from flask.cli import with_appcontext
 from grc.models import db, Application, ApplicationStatus
 from grc.utils.application_files import ApplicationFiles
+from grc.utils.logger import LogLevel, Logger
 
+logger = Logger()
```

Comment:
I wouldn't use the logger in the jobs. The logger uses the flask session data and jobs won't be in a session so it will complain about working outside the app context. I'd revert these back to print statements in ALL the jobs.